### PR TITLE
Try to auto lookup doc if there is no selection

### PR DESCRIPTION
**Pull request by jfcherng**
If there is no selected region, we look up the doc by user's cursor position automatically.

### CODE REVIEW

**Comment by baptisteArno** [comment]
I'll run tests and get back to you today. Thanks!

File: sublime_tensorflow_autocomplete.py [hunk 1]
```diff
@@ -2090,13 +2090,38 @@ class TensorflowDocCommand(sublime_plugin.TextCommand):
...
+        else:
+            for region in self.view.sel():
+                selection += self.view.substr(region)
 
         if selection + "()" in tensorflow_functions:
+            selec_link = selection.replace('.', '/')
             webbrowser.open("https://www.tensorflow.org/api_docs/python/%s" % selec_link)
-        else:
-            sublime.error_message("Not a Tensorflow class or function.\nHere is an example of what can be selected : 'tf.nn.conv2d'")
+        elif not is_auto_selection:
```
Comment (baptisteArno) [inline sublime_tensorflow_autocomplete.py]: This means the error message will only be displayed when there is a selection. It's preferable to be displayed even if there is no selection?

**Review by baptisteArno — DISMISSED** [review]

File: sublime_tensorflow_autocomplete.py (hunk 1 above)
Comment (jfcherng) [reply sublime_tensorflow_autocomplete.py]: Agree.

**Review by baptisteArno — APPROVED** [review]
Your changes are perfect!

**Pull request merged**